README.md

```diff
@@ -48,6 +49,38 @@ curl --request POST \
 
 5) Open [http://localhost:8080](http://localhost:8080) to see your uploaded docs !
 
+### Delete a doc
+
+1) Launch the application and its server with ```docker run -e CREDENTIALS=user:password -e SHOULD_SECURE=FALSE -v `pwd`:/data -p 8080:80 tracesoftware/hostmydocs```
+
+2) delete using cURL by example :
+```bash
+curl --request DELETE \
```

Comment:
This command can be simplified

Reply:
In you case, I think some parameters can be omitted :wink:

Reply:
command was simplified